Marlin/Configuration_adv.h

```diff
@@ -1531,8 +1531,9 @@
       #define BOOTSCREEN_TIMEOUT 4000       // (ms) Total Duration to display the boot screen(s)
       #if EITHER(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
         #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
+        //#define BOOT_MARLIN_LOGO_ANIMATED   // Choose either small static logo, large animated logo, or BOTH shows small animated logo
```

Comment:
`TFT_COLOR_UI` does not have an animated boot screen, so `BOOT_MARLIN_LOGO_ANIMATED` does not apply.

Reply:
well I tried. at least the cleanup for stm32f4.ini is something